src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Proxy.cs

```diff
@@ -0,0 +1,302 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Test.Common;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP HTTP stack doesn't support .Proxy property")]
+    public abstract class HttpClientHandler_Proxy_Test : HttpClientTestBase
+    {
+        private readonly ITestOutputHelper _output;
+        
+        public HttpClientHandler_Proxy_Test(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+        
+        [OuterLoop("Uses external server")]
+        [Theory]
+        //[ActiveIssue(30327)]
+        //[InlineData(AuthenticationSchemes.Ntlm, false, true)]
+        //[InlineData(AuthenticationSchemes.Negotiate, false, true)]
+        //[InlineData(AuthenticationSchemes.Ntlm, true, true)]
+        //[InlineData(AuthenticationSchemes.Negotiate, true, true)]
```

Comment:
Why are these commented out?

Reply:
Because of the ActiveIssue above. I thought it was useful to leave these cases here with the activeissue.  It is the next bug to fix and enable these specific cases.

Reply:
I see.  I didn't understand they were related (I assumed you were commenting out the ActiveIssue as part of the PR to test that things actually worked).

Reply:
I'll think I'll remove these commented out InlineData to reduce confusion.